govcd/openapi_org_network.go

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright 2021 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// OpenApiOrgVdcNetwork uses OpenAPI endpoint to operate both - NSX-T and NSX-V Org VDC networks
+type OpenApiOrgVdcNetwork struct {
+	OpenApiOrgVdcNetwork *types.OpenApiOrgVdcNetwork
+	client               *Client
```

Comment:
Private/lower `client` case on purpose?

Reply:
Yes. In fact we have it so in many/almost all places. VM, vApp, VDC, etc...